### ISSUE TITLE
Make it obvious if an allele of a genotype has a "comment"

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#2545]
I was looking for a few of these to fix, and they are tricky to locate in big sessions with lots of similar alleles. 
I suggest a small link in the genotype panel (i.e like the one we plan to add to the phenotype panel to access the GIs)

It would also make it more obviups if a comment existed when I fied some alleles. For example, I missed this one:
llele numbering PMID:15173383  are based on the Pignede sequence (off by 2)	44ae5ece389a1b0a	cdc6-C1043A(C1043A)[Wild_type]
Allele numbering PMID:15173383  are based on the Pignede sequence (off by 2)	44ae5ece389a1b0a	cdc6-C1058S(C1058S)[Wild_type]
Allele numbering PMID:15173383  are based on the Pignede sequence (off by 2)	44ae5ece389a1b0a	cdc6-C1053A(C1053A)[Wild_type]
Allele numbering PMID:15173383  are based on the Pignede sequence (off by 2)	44ae5ece389a1b0a	cdc6-C1058A(C1058A)[Wild_type]
Allele numbering PMID:15173383  are based on the Pignede sequence (off by 2)	44ae5ece389a1b0a	cdc6-C1040A(C1040A)[Wild_type]

when I aligned these alleles with the current sequence- these are now synonyms because the same allele is named this way in other papers ( I left the comment because it is useful)

**Comment by ValWood** [comment]
Low priority because there aren't many and I can see/review  them all in the 2 files

**Comment by kimrutherford** [comment]
Just to be clear: these are genotype comments not allele comments/notes

**Comment by ValWood** [comment]
Yep, I need to review them and move many of them over to allele comments one those are visible.
Most of them probably apply to alleles.
